src/components/Tooltip/partials/BrightHeader.tsx

```diff
@@ -0,0 +1,47 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import React, { FC, ReactElement } from "react";
+import { merge } from "@utilities/merge";
+import { IconSize } from "@elements/Icon/IconSize";
+import IconInfo from "@elements/Icon/Generated/IconInfo";
+import IconCheck from "@elements/Icon/Generated/IconCheck";
+import IconCallout from "@elements/Icon/Generated/IconCallout";
+import IconDocument from "@elements/Icon/Generated/IconDocument";
+
+export enum BrightHeaderVariants {
```

Comment:
We haven't used variants in the other components, so I'd suggest renaming this to `BrightHeaderStyle` to keep it more consistent and capitalise the values (see e.g. `BadgeStyle`)